controls/V-71973.rb

```diff
@@ -60,39 +40,56 @@
 
 # yum list installed aide
 
-If AIDE is not installed, ask the SA how file integrity checks are performed on the
-system.
+If AIDE is not installed, ask the SA how file integrity checks are performed on
+the system.
 
 Check for the presence of a cron job running daily or weekly on the system that
-executes AIDE daily to scan for changes to the system baseline. The command used in
-the example will use a daily occurrence.
+executes AIDE daily to scan for changes to the system baseline. The command
+used in the example will use a daily occurrence.
 
-Check the \"/etc/cron.daily\" subdirectory for a \"crontab\" file controlling the
-execution of the file integrity application. For example, if AIDE is installed on
-the system, use the following command:
+Check the \"/etc/cron.daily\" subdirectory for a \"crontab\" file controlling
+the execution of the file integrity application. For example, if AIDE is
+installed on the system, use the following command:
 
 # ls -al /etc/cron.* | grep aide
 -rwxr-xr-x  1 root root        29 Nov  22  2015 aide
 
-If the file integrity application does not exist, or a \"crontab\" file does not
-exist in the \"/etc/cron.daily\" or \"/etc/cron.weekly\" subdirectories, this is a
-finding."
-  tag "fix": "Configure the file integrity tool to automatically run on the system
-at least weekly. The following example output is generic. It will set cron to run
-AIDE daily, but other file integrity tools may be used:
+If the file integrity application does not exist, or a \"crontab\" file does
+not exist in the \"/etc/cron.daily\" or \"/etc/cron.weekly\" subdirectories,
+this is a finding."
+  tag "fix": "Configure the file integrity tool to automatically run on the
+system at least weekly. The following example output is generic. It will set
+cron to run AIDE daily, but other file integrity tools may be used:
 
 # cat /etc/cron.daily/aide
-0 0 * * * /usr/sbin/aide --check | /bin/mail -s \"aide integrity check run for
-<system name>\" root@sysname.mil"
-
-  describe package(file_integ_tool) do
+0 0 * * * /usr/sbin/aide --check | /bin/mail -s \"aide integrity check run for <system name>\" root@sysname.mil"
+  tag "fix_id": "F-78325r1_fix"
+  describe package(FILE_INTEGRITY_TOOL) do
     it { should be_installed }
   end
-  describe.one do
-    describe file("/etc/cron.daily/#{file_integ_tool}") do
-      it { should exist }
+  if FILE_INTEGRITY_INTERVAL == 'monthly'
```

Comment:
This can also be in root's crontab and/or `/etc/cron.d` so those should also be checked